examples/flowcam.py

```diff
@@ -12,9 +12,11 @@
 from morphocut.stream import TQDM
 
 import_path = "../tests/data/flowcam"
-export_path = "../tmp/flowcam"
 
-os.makedirs(export_path, exist_ok=True)
+file_path = __file__
+file_name = os.path.basename(file_path)
+out_folder = file_name.split('.')[0] + "_out"
+os.makedirs(out_folder, exist_ok=True)
```

Comment:
While clever, this makes the example unnecessarily hard to understand for novices.